x-pack/plugins/cloud_security_posture/public/pages/rules/rules_container.tsx

```diff
@@ -101,9 +139,32 @@ export const RulesContainer = () => {
     () => allRules.data?.items.map((rule) => rule.metadata.benchmark.rule_number || ''),
     [allRules.data]
   );
-  const cleanedSectionList = [...new Set(sectionList)];
+  const cleanedSectionList = [...new Set(sectionList)].sort((a, b) => {
+    return a.localeCompare(b, 'en', { sensitivity: 'base' });
+  });
   const cleanedRuleNumberList = [...new Set(ruleNumberList)];
 
+  const rulesPageData = useMemo(
+    () => getRulesPage(filteredRulesWithStatuses, status, error, rulesQuery),
+    [filteredRulesWithStatuses, status, error, rulesQuery]
+  );
+
+  const [selectedRules, setSelectedRules] = useState<CspBenchmarkRulesWithStatus[]>([]);
+
+  const setSelectAllRules = () => {
+    setSelectedRules(rulesPageData.all_rules);
+  };
+
+  const rulesFlyoutData: CspBenchmarkRulesWithStatus = {
+    ...{
+      status:
+        filteredRulesStates.find((e) => e.rule_id === selectedRuleId)?.muted === true
```

Comment:
```suggestion
        filteredRulesStates.find((filteredRulesState) => filteredRulesState.rule_id === selectedRuleId)?.muted === true
```

`e` doesn't mean much.